docs/whats_new.rst

```diff
@@ -18,6 +18,15 @@ Breaking Changes
 
 Enhancements
 ~~~~~~~~~~~~
+- Works with unstructured 1D grids such as:
+
+  - `ICON <https://code.mpimet.mpg.de/projects/iconpublic>`_
+  - `FESOM <https://fesom.de/>`_
+  - `MPAS <https://mpas-dev.github.io/>`_
+
+  with 1-dimensional coordinates of the form ``lon(cell)`` and ``lat(cell)``.
+  Note that only xarray arrays can be detected as unstructured grids.
+  (:issue:`278`, :pull:`280`). By `Aaron Spring <https://github.com/aaronspring>`_.
```

Comment:
what about `wrap_lon=False`, should that be specified here also? @mathause

Reply:
That should no longer be necessary with 

https://github.com/regionmask/regionmask/blob/33bf93778e1a20b1b903a8f8f54a13d9ecbc23be/regionmask/core/utils.py#L103

- or do I misunderstand something?

Reply:
the error appears later in https://github.com/regionmask/regionmask/blob/33bf93778e1a20b1b903a8f8f54a13d9ecbc23be/regionmask/core/mask.py#L184

Reply:
adding `and not is_unstructured` in
https://github.com/regionmask/regionmask/blob/33bf93778e1a20b1b903a8f8f54a13d9ecbc23be/regionmask/core/mask.py#L182

Reply:
Ah yes, good we test all that. However, I prefer if you thread `is_unstructured` trough `_mask_edgepoints_shapely` to

https://github.com/regionmask/regionmask/blob/33bf93778e1a20b1b903a8f8f54a13d9ecbc23be/regionmask/core/mask.py#L385

(instead of L182) for consistency.